docs/spfx/build-for-teams-configure-in-teams.md

```diff
@@ -0,0 +1,69 @@
+
+## Configure Microsoft Teams personal apps built using SharePoint Framework
+
+Microsoft Teams personal apps don’t offer any infrastructure for implementing configuration. Instead, following the pattern recommended by Microsoft Teams, personal app’s [settings should be exposed in a separate tab](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/design/personal-apps).
+
+:::image type="content" source="../images/build-for-teams/build-for-teams-personal-app.png" alt-text="Personal app with multiple tabs including settings built using SharePoint Framework":::
+
+Translating this to your SharePoint Framework solution, it means building a separate web part that contains the configuration UI and which will be used only in the context of the personal app, defining a storage for user’s configuration and extending the personal app definition to contain multiple tabs.
+
+### Personal app configuration UI web part
+
+Each tab in a personal app is mapped to a URL. The easiest way to build UI for configuring your SharePoint Framework web part exposed as a personal app, is by building a separate web part with the configuration UI. This will allow you to optimize the UI for use in the context of the personal app and keep this code separate from your web part.
+
+Since the personal app configuration UI web part is not meant to be used outside of the personal app, in its manifest, you should set the **supportedHosts** property to an empty array.
+
+:::image type="content" source="../images/build-for-teams/build-for-teams-manifest-setttings-webpart.png" alt-text="The supportedHosts property of a SharePoint Framework web part used as a personal app configuration UI":::
+
+The reason you want the **supportedHosts** property to be empty is to prevent the web part from being used in SharePoint but also not include it in the autogenerated Teams manifest. By default, each web part that contains the **TeamsTab** or **TeamsPersonalApp** value in the manifest is included in the generated Teams manifest as a separate Microsoft Teams app. In this case however, you want the personal app to consist of multiple tab, each pointing to a different web part. This can be done only by manually updating the manifest yourself.
+
+To add a tab to your personal app and have it point to another web part, in the Teams manifest defined in the **teams/manifest.json** file, navigate to the **staticTabs** section and copy the existing entry. In the copied entry, update values of the **entityId** and **name** properties. In the **contentUrl** property, update the value of the **componentId** query string parameter so that it matches the ID of your settings web part as defined in its manifest.
```

Comment:
Same comment as previous about the inline code formatting for all values in this paragraph, except the file **teams/manifest.json**. That should remain bold.